api/src/main/java/com/cloud/agent/api/to/VirtualMachineTO.java

```diff
@@ -25,14 +25,13 @@
 import com.cloud.template.VirtualMachineTemplate.BootloaderType;
 import com.cloud.vm.VirtualMachine;
 import com.cloud.vm.VirtualMachine.Type;
```

Comment:
```suggestion
import com.cloud.vm.VirtualMachine.Type;

```
Adding the removed line back